pkg/cmd/join/exec.go

```diff
@@ -114,7 +119,39 @@ func (o *Options) run() error {
 		return err
 	}
 	output = append(output, out...)
-	fmt.Printf("Deploying klusterlet agent. Please wait a few minutes then log onto the hub cluster and run the following command:\n\n"+
+
+	timeout := int64(300)
+	gvr := schema.GroupVersionResource{Group: "operator.open-cluster-management.io", Version: "v1", Resource: "klusterlets"}
```

Comment:
I may ask why not use klusterlet client...

Reply:
Oh, that could be easier in use than the dynamic client. Let me try it out.